presto-docker/bin/entrypoint

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+# Copy config files, if they exist, to /usr/lib/presto/etc. This is necessary
+# for Kubernetes as Secrets and ConfigMaps are mounted read only.
+if [[ -d /usr/lib/presto/conf ]]; then
+    # Delete all default config files including catalogs
+    find /usr/lib/presto/etc/* -type f -delete
+    cp -r /usr/lib/presto/conf/* /usr/lib/presto/etc/
```

Comment:
shall we use mounted properties instead of copying them?

if copying file from `/usr/lib/presto/conf/*` to `/usr/lib/presto/etc/` within this script, it means such process will happen once only during pod initialization. feature like [refresh ACL properties file without restart presto](https://prestosql.io/docs/current/security/built-in-system-access-control.html#refresh) will break (a container does receive configmap updates as long as it doesn't use `subPath` under volumeMounts).

My use case is using helm chart maintain presto cluster. when updating ACL properties via `helm upgrade`, running containers will receive configmap changes so that I don't need to kill all presto pods.

Reply:
That's a good point, I wasn't aware of the auto-refresh feature for ACLs. However as ConfigMap, Secrets, etc. are mounted read only and our need to update `node.properties` at container start time this is impractical with the current setup. I have considered using something like [confd](https://github.com/kelseyhightower/confd) to handle issues like this, but at the time of writing I didn't know of a use case which required it. I'm inclined to fix the problem you've presented, but I'd like to push it to the next version of this `Dockerfile` in order to survey other possible needs which can be served by something like confd.

Reply:
it's totally fine to push such design requirement to next version of `Dockerfile`.

I can comment on #396 to share how we generate presto configuration files prior to container start via k8s helm charts.

Reply:
Actually @shawnzhu while talking over an unrelated problem I actually have a pretty good fix for this coming with my next push. It's not using confd, but it is switching around how the defaults and mounting happen so that it will be possible to mount read-only config.